Sources/BraintreeShopperInsights/BTShopperInsightsClientV2.swift

```diff
@@ -43,6 +43,17 @@ public class BTShopperInsightsClientV2 {
         return try await createCustomerSessionAPI.execute(request)
     }
     
+    /// This method updates an existing customer session.
+    /// - Parameters:
+    ///    - request: A `BTCustomerSessionRequest`
+    ///    - sessionID: the ID of the session to update
+    /// - Returns: A `String` representing a session ID if successful
+    /// - Throws: An error if the request fails for some reason or if the response is invalid.
```

Comment:
Can we add the beta warning to this and the `createCustomerSession` method? Just noticed those were missing.